### PR TITLE
CRDB namespaces updates

### LINKED ISSUE #7272: sql: add user-defined schemas/name resolution in pg compat page
Developers regularly ask our sales team if we support user-defined schemas (also known as `create schema`) in Postgres. We hope to add this feature in the next release but in the meantime we should make it easier to discover that you can use `databases` in CockroachDB like schemas in Postgres. We have an excellent [name resolution docs page](https://www.cockroachlabs.com/docs/v19.2/sql-name-resolution.html) but it doesn't quite make it clear that you can use the info within to address the lack of `create schema` and we don't link this page from out [Postgres compatibility page](https://www.cockroachlabs.com/docs/stable/postgresql-compatibility.html).

I propose the following plan:

- [ ] Update name resolution page with an explicit call out that you can use CockroachDB databases as a workaround to no create schema 
- [ ] Update pg compat page to include this same language and link to the name resolution
- [ ] Make these changes for 20.1, 19.2, and 19.1 documentation

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes #7272.

- Updated Name Resolution page for clarity, formatting, and accuracy.
- Added section on schema namespaces to PostgreSQL Compatibility page.

### CODE REVIEW

**Comment by ericharmeling** [comment]
@awoods187 
Friendly ping on review :)

**Comment by awoods187** [comment]
This LGTM. I'd like @vy-ton and @solongordon to PTAL

**Review by arulajmani — APPROVED** [review]
:lgtm: 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/8493#-:-MJ9EH6_AxM_xgfbpHj3:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @awoods187, @solongordon, and @vy-ton)

**Review by vy-ton — APPROVED** [review]

File: v20.2/sql-name-resolution.md [hunk 1]
```diff
@@ -4,30 +4,27 @@ summary: Table and function names can exist in multiple places. Resolution decid
...
 
-<span class="version-tag">New in v20.2</span>: CockroachDB also supports user-defined schemas with the [`CREATE SCHEMA` statement](create-schema.html).
+In addition to the `public` schema, CockroachDB supports a fixed set of virtual schemas, available in every database, that provide ancillary, non-stored data to client applications. For example, [`information_schema`](information-schema.html) is provided for compatibility with the SQL standard.
 
-The list of all databases can be obtained with [`SHOW DATABASES`](show-databases.html). The list of all schemas for a given database can be obtained with [`SHOW SCHEMAS`](show-schemas.html). The list of all objects for a given schema can be obtained with other `SHOW` statements.
+To create a new database, use a [`CREATE DATABASE`](create-database.html) statement. To create a new schema, use a [`CREATE SCHEMA`](create-schema.html) statement. The list of all databases can be obtained with [`SHOW DATABASES`](show-databases.html). The list of all schemas for a given database can be obtained with [`SHOW SCHEMAS`](show-schemas.html). The list of all objects for a given schema can be obtained with other `SHOW` statements.
```
Comment (vy-ton) [inline v20.2/sql-name-resolution.md:27]: Can we link to the `Convert to Schema` page?

**Review by lnhsingh — COMMENTED** [review]
:lgtm: 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/8493#-:-MJTo_2H3dEYl2iOiHOL:bg7igkv)* status: :shipit: complete! 2 of 0 LGTMs obtained (waiting on @awoods187, @ericharmeling, and @lnhsingh)

**Comment by ericharmeling** [comment]
TFTR @lnhsingh !

**Pull request merged**